modules/acquia_cms_common/acquia_cms_common.module

```diff
@@ -437,6 +437,13 @@ function acquia_cms_common_modules_installed($modules) {
       \Drupal::service('cohesion_sync.package_import_handler')->importPackagesFromPath($packages);
     }
   }
+
+  // Acquia connector check and set telemetry opt-in feature.
+  if (\Drupal::moduleHandler()->moduleExists('acquia_connector') &&
```

Comment:
@apathak18 I think there's no need to check this on `hook_modules_installed()` instead we should add this in `hook_install()`.